hypershift-operator/controllers/nodepool/haproxy.go

```diff
@@ -221,11 +208,10 @@ type fileToAdd struct {
 	source   func() ([]byte, error)
 	name     string
 	mode     int
-	params   map[string]string
+	params   map[string]any
```

Comment:
why does this need any now?

Reply:
because of the new bool var `UseProxyProtocol`, passing it as a string "true" for example would require us to do string equality in the template instead of just an if statement. Also we don't have to parse the ports integers.